static/app/views/alerts/rules/issue/setupMessagingIntegrationButton.tsx

```diff
@@ -9,28 +9,26 @@ import type {
   IntegrationProvider,
   OrganizationIntegration,
 } from 'sentry/types/integrations';
-import {trackAnalytics} from 'sentry/utils/analytics';
 import {getIntegrationFeatureGate} from 'sentry/utils/integrationUtil';
 import {useApiQueries, useApiQuery} from 'sentry/utils/queryClient';
 import useOrganization from 'sentry/utils/useOrganization';
 import MessagingIntegrationModal from 'sentry/views/alerts/rules/issue/messagingIntegrationModal';
 
 export enum MessagingIntegrationAnalyticsView {
-  ALERT_RULE_CREATION = 'alert_rule_creation',
-  PROJECT_CREATION = 'project_creation',
+  ALERT_RULE_CREATION = 'alert_rule_creation_messaging_integration_onboarding',
+  PROJECT_CREATION = 'project_creation_messaging_integration_onboarding',
+  DEFAULT = 'messaging_integration_onboarding',
 }
 
 type Props = {
-  analyticsParams?: {
-    view: MessagingIntegrationAnalyticsView;
-  };
+  analyticsView: MessagingIntegrationAnalyticsView;
   projectId?: string;
   refetchConfigs?: () => void;
 };
 
 function SetupMessagingIntegrationButton({
   refetchConfigs,
-  analyticsParams,
+  analyticsView,
```

Comment:
If that's the default value, could we use it here as the default? Not sure if we _always_ want to use that as the default though (not enough context)
```suggestion
  analyticsView = MessagingIntegrationAnalyticsView.DEFAULT,
```